code/game/objects/items/devices/scanners.dm

```diff
@@ -153,8 +153,11 @@ REAGENT SCANNER
 	if(H.timeofdeath && (H.stat == DEAD || (HAS_TRAIT(H, TRAIT_FAKEDEATH))))
 		to_chat(user, "<span class='notice'>Time of Death: [station_time_timestamp("hh:mm:ss", H.timeofdeath)]</span>")
 		var/tdelta = round(world.time - H.timeofdeath)
-		if(tdelta < DEFIB_TIME_LIMIT && !DNR)
+		if(H.under_defib_timer() && !DNR)
 			to_chat(user, "<span class='danger'>Subject died [DisplayTimeText(tdelta)] ago, defibrillation may be possible!</span>")
+			if(advanced)
+				var/time_left = round(H.defib_time_left() / 10)
+				to_chat(user, "<span class='red'>Estimated time before unrecoverable tissue damage: [DisplayTimeText(time_left)].</span>")
```

Comment:
maths teacher be like:
```suggestion
				to_chat(user, "<span class='red'>Estimated time before unrecoverable tissue damage: [DisplayTimeText(time_left)] seconds.</span>")
```

Reply:
DisplayTimeText should display it in minutes and seconds already.